test/commands/solution/search_community_solutions_test.rb

```diff
@@ -161,9 +161,12 @@ class Solution::SearchCommunitySolutionsTest < ActiveSupport::TestCase
   test "filter: sync_status" do
     track = create :track
     exercise = create :concept_exercise, track: track
-    solution_1 = create :concept_solution, :published, exercise: exercise, git_important_files_hash: exercise.git_important_files_hash
-    solution_2 = create :concept_solution, :published, exercise: exercise, git_important_files_hash: exercise.git_important_files_hash
-    solution_3 = create :concept_solution, :published, exercise: exercise, git_important_files_hash: 'different_hash'
+    solution_1 = create :concept_solution, exercise: exercise, git_important_files_hash: exercise.git_important_files_hash,
+num_stars: 11, published_at: Time.current, status: :published
+    solution_2 = create :concept_solution, exercise: exercise, git_important_files_hash: exercise.git_important_files_hash,
+num_stars: 22, published_at: Time.current, status: :published
+    solution_3 = create :concept_solution, exercise: exercise, git_important_files_hash: 'different_hash', num_stars: 33,
+published_at: Time.current, status: :published
```

Comment:
Can you indent these pls.

Reply:
I did. Rubocop changed them back. I'll force it